### ISSUE TITLE
Feature: Add database to store requests and their status

### ISSUE THREAD

**Issue by romaraOM** [comaraDOTcom/flask-demo#1]
It would be very useful if we had some backend database which could store the incoming requests from users and create a database of these requests, with statuses/requests/timestamps etc.

**Comment by comaraDOTcom** [comment]
Yes this would be a useful feature to add. Would a [sql3lite](https://docs.python.org/2/library/sqlite3.html) client be the best way to add to this?
